pkg/controller/storagecluster/reconcile.go

```diff
@@ -277,26 +260,6 @@ func newCephCluster(sc *ocsv1alpha1.StorageCluster) *rookCephv1.CephCluster {
 						},
 					},
 				},
-				"osd": rook.Placement{
```

Comment:
Yes. Will be using `Placement` Spec under StorageClassDeviceSet for PVC based OSDs to be placed. Still waiting on Labels to be added to these resources so that selectors can be used for configurations.

Few things I am unclear about is the topologyKey. Do we use host or AZ for spreading these OSDs? Or, do we use both (if we can)?

Do we use `PreferredDuringScheduling` or `RequiredDuringScheduling`? `Preferred` sounds like a better option to me. What would be the appropriate value for `weight` ?

Reply:
I hv added placement to StorageClassDeviceSet internally. But, I see that StorageCluster CR exposes this placement config to user. Do you think that is a good idea? Specially when we are forcing particular labels (https://github.com/openshift/ocs-operator/blob/master/deploy/olm-catalog/ocs-operator/0.0.1/ocs-operator.v0.0.1.clusterserviceversion.yaml#L9-L41).